backend/prisma/schema.prisma

```diff
@@ -5,9 +5,11 @@
 // Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
 
 generator client {
-  provider = "prisma-client-js"
+  provider      = "prisma-client-js"
+  binaryTargets = ["native"]
```

Comment:
Is this necessary

Reply:
It works without it but in the prisma documentation and forums, it is recommended as good practice
https://www.prisma.io/docs/orm/prisma-schema/overview/generators

Reply:
okay